onadata/apps/api/viewsets/project_viewset.py

```diff
@@ -91,6 +92,13 @@ def forms(self, request, **kwargs):
 
                 serializer = serializer_cls(survey,
                                             context={'request': request})
+
+                published_by_formbuilder = request.data.get(
+                    'published_by_formbuilder'
+                )
+                if published_by_formbuilder == u'True':
```

Comment:
Where do I find it? Also, `published_by_formbuilder` has string value, how will that utility function be helpful?